src/NugetSupportFiles/Microsoft.Diagnostics.Tracing.TraceEvent.SupportFiles.Populate.bat

```diff
@@ -1,5 +1,5 @@
 REM copies from an existing build to a nuget package creation area (so that *.MakeNuget.bat works)
-xcopy /s ..\PerfView\packages\Microsoft.Diagnostics.Tracing.TraceEvent.SupportFiles.1.0.1\*.dll Microsoft.Diagnostics.Tracing.TraceEvent.SupportFiles
+xcopy /s ..\..\packages\Microsoft.Diagnostics.Tracing.TraceEvent.SupportFiles.1.0.1\*.dll Microsoft.Diagnostics.Tracing.TraceEvent.SupportFiles
```

Comment:
Well, at the time this was used, there did not exist a 1.0.2 to copy from (that was what was being created).    Still I can update it now, and I added some more comments / docs to make it less likely people mess this up.